src/Components/NewClient.js

```diff
@@ -1,32 +1,101 @@
-import React from "react";
+import React, { useState } from "react";
 import "./NewClient.css";
 
-class NewClient extends React.Component {
-  render() {
-    return (
-      <section className="new-client">
+export default function NewClient() {
+  // const [ firstName, setFirstName ] = useState("");
+  // const [ lastName, setLastName ] = useState("");
+  // const [ phone, setPhone ] = useState("");
+  // const [ email, setEmail ] = useState("");
```

Comment:
Turning all of these into one object in state was a great move. Your single `handleUserInput` event handler is also very clean and concise. Nice job!